src/core/qgsdiagramrendererv2.h

```diff
@@ -340,7 +342,22 @@ class CORE_EXPORT QgsDiagramSettings
     //! @note added in 2.10
     QList< QString > categoryLabels;
     QSizeF size; //size
-    SizeType sizeType; //mm or map units
+    /** Diagram size unit index (mm, map units, or pixels)
+     * @note added in 2.16
+     */
+    QgsSymbolV2::OutputUnit sizeType;
```

Comment:
Very minor thing - but the sip binding docs are different here. It'll cause more work for Jürgen when he next syncs the bindings ;) 

(I'd also drop the "mm, map units, pixels" part as there's always a change that other units will be added in future!)